build.yaml

```diff
@@ -1435,6 +1396,46 @@ job_type:
             ]
           }
       timeout: 120
+    run_client_installation_on_OSX:
+      at: '0 6 * * *'
```

Comment:
This is the only added line, the rest is the context, it has moved from
one block to the cronly_jobs block, which means that jobs.groovy
will run it nightly on master.